components/Navigation/Navigation.js

```diff
@@ -0,0 +1,55 @@
+import {
+  Footer,
+  Nav,
+  NavLink,
+  MapIcon,
+  RouteIcon,
+  StarIcon,
+  BikeIcon,
+} from "./StyledNavigation";
+
+export default function Navigation({
+  activePage,
+  onMapIconClick,
+  onRouteIconClick,
+  onStarIconClick,
+  onBikeIconClick,
+}) {
+  return (
+    <Footer>
+      <Nav>
+        <NavLink
+          isactive={activePage === "Map" ? "on" : "off"}
+          onClick={onMapIconClick}
```

Comment:
you can remove all onClick events in this file after implementing my suggestion :) 

```suggestion
```